### ISSUE TITLE
Performance about matterport

### ISSUE THREAD

**Issue by congwei45** [pengsongyou/openscene#53]
I follow your config when training on matterport, but get mIoU=41.4% and mAcc=55.9, which is lower than the reported mIoU=42.6 and mAcc=59.2.

**Comment by pengsongyou** [comment]
Have you tried to directly use the pretrained weights? Can you reproduce
our results reported in the paper with that?

**Comment by congwei45** [comment]
When using the pretrained weights, the mIoU=41.8 and the mAcc=58.8.

**Comment by pengsongyou** [comment]
Have you run the evaluation repeatedly for 5 times？Basically [here](https://github.com/pengsongyou/openscene/blob/main/config/matterport/ours_openseg_pretrained.yaml#L45), and the code [here](https://github.com/pengsongyou/openscene/blob/main/run/evaluate.py#L414-L426). If you only run the evaluation with `test_repeat: 1`, the results are probably lower than what is reported in the paper. We set `test_repeat: 5` because it is a common trick used in previous works.

**Comment by congwei45** [comment]
Why is the accuracy of the model I trained myself much lower than the one you provided? Can you help me analyze the reason?

**Comment by pengsongyou** [comment]
First of all, based on my own testing, if you follow the instructions exactly as discussed, you should be able to reproduce the results. You might change something minor in the process (e.g. version of PyTorch or so), this might make a difference. 

Second, your result was indeed lower, but I would not say "much lower", since your mIoU is just 1.2% lower. Can you try changing the random seed, which might make a difference as well. Also, make sure you use `test_repeat: 5` in your config.
